samples/buddy-auth/src/buddy_auth/service.clj

```diff
@@ -0,0 +1,86 @@
+(ns buddy-auth.service
+  "A simple service demonstrating integration with the buddy-auth
+  authentication/authorization library."
+  (:require [io.pedestal.http :as http]
+            [io.pedestal.http.route :as route]
+            [io.pedestal.http.body-params :as body-params]
+            [io.pedestal.interceptor.chain :as interceptor.chain]
+            [io.pedestal.interceptor.error :refer [error-dispatch]]
+            [io.pedestal.interceptor.helpers :refer [on-request]]
+            [ring.util.response :as ring-resp]
+            [buddy.auth :as auth]
+            [buddy.auth.backends :as auth.backends]
+            [buddy.auth.middleware :as auth.middleware]))
+
+(def users
+  "A sample user store."
+  {:aaron {:display-name "Aaron Aardvark"
+           :password     "secret"
+           :role         :user}
+   :gmw   {:display-name "Gerald M. Weinberg"
+           :password     "rutabaga"
+           :role         :admin}})
+
+(def basic-auth-backend
+  "A buddy-auth Basic Authentication backend.  See
+  https://funcool.github.io/buddy-auth/latest/#http-basic"
+  (auth.backends/basic {:realm  "MyApi"
+                        :authfn (fn [_ authdata]
```

Comment:
Could you replace the unused arg `_` with something named, just so new users have some context what that arg is for?  Thanks